x-pack/solutions/security/plugins/security_solution/public/flyout/attack_details/index.tsx

```diff
@@ -33,28 +32,28 @@ export const AttackDetailsPanel: React.FC<Partial<AttackDetailsProps>> = memo(({
 
   const { tabsDisplayed, selectedTabId } = useTabs({ path });
 
-  const setSelectedTabId = (tabId: AttackDetailsPanelTabType['id']) => {
-    openRightPanel({
-      id: AttackDetailsRightPanelKey,
-      path: { tab: tabId },
-      params: { attackId },
-    });
-    // saving which tab is currently selected in the right panel in local storage
-    storage.set(FLYOUT_STORAGE_KEYS.RIGHT_PANEL_SELECTED_TABS, tabId);
-  };
+  const setSelectedTabId = useCallback(
+    (tabId: AttackDetailsPanelTabType['id']) => {
+      openRightPanel({
+        id: AttackDetailsRightPanelKey,
+        path: { tab: tabId },
+        params: { attackId },
```

Comment:
Are we suppose to path `indexName` of the attack here? If not, where exactly are we passing the attack's index to the context?